.changeset/eight-oranges-appear.md

```diff
@@ -0,0 +1,7 @@
+---
+"saleor-app-search": minor
+---
+
+Extended search model with two new fields:
+- `inStock`: returns true if stock is available. To keep this up to date, the application subscribes to `ProductVariantOutOfStock` and `ProductVariantBackInStock` events
```

Comment:
hmm we need to add these events. I think we can inform that it requires a reinstall? Otherwise we need a full blown migration. Its more than adding and fetching new fields, we actually need to register these at some point in time